lib/sealField.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const iron = require('iron');
+const nodeify = require('./nodeify');
+
+/**
+ * Seal a field
+ * @method sealField
+ * @param  {Object}         object              The object to seal
+ * @param  {Object}         config
+ * @param  {Mixed}          config.field        The field to store the sealed value in
+ * @param  {String}         [config.value]      The value to seal. Defaults to the current value of the field
```

Comment:
Was thinking about the case where you set a field to an unsealed value, called seal, iron errored, and then called update. Wouldn't want to save the unsealed value in the datastore.

Though now I realize that no code should just blindly be calling update, if iron fails it should go to a catch block of some sort.